src/test/java/org/jpeek/MetricsTest.java

```diff
@@ -70,6 +70,8 @@
  * @checkstyle VisibilityModifierCheck (500 lines)
  * @checkstyle JavadocVariableCheck (500 lines)
  * @checkstyle MagicNumberCheck (500 lines)
+ * @todo #67:30min PCC: add the rest of the test cases for this metric. Could
+ *  only fit test case for MethodsWithDiffParamTypes within budget.
```

Comment:
@llorllale Is there a bug in the implemented metric? Why did you not have time to add the other classes? I imagine that simply adding a few rows here does not take a lot of time so the tests must have failed and you did not have time to fix them?

Reply:
@amihaiemil it simply means that all that work could not fit inside the 30min budget allotted for this task. "all that work" in this case means doing the math for each test case and validating them by running the tests.

Reply:
@llorllale right, I see. So you have to calculate the outcome manually before writing the tests. Sorry, my bad...

Reply:
@amihaiemil yeah, otherwise it would be cheating :)

It would've been great if we had some other software that could automate that for us but the lack of said software is the reason for this project in the first place :)